catalog-entities/components/showcase.yaml

```diff
@@ -16,9 +19,17 @@ metadata:
     backstage.io/kubernetes-id: "janus-idp"
     github.com/project-slug: janus-idp/backstage-showcase
     quay.io/repository-slug: janus-idp/backstage-showcase
-    backstage.io/techdocs-ref: dir:.
+    backstage.io/techdocs-ref: url:https://github.com/janus-idp/backstage-showcase
     backstage.io/kubernetes-namespace: janus-idp
```

Comment:
It's missing the SonarQube annotation
`sonarqube.org/project-key: janus-idp_backstage-showcase`